tests/models/user.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import pytest
+
+import accounts
+
+from .store import _update_from_upload_file
+
+
+def _create_submission_and_suggestion(store, user):
+    # TODO: test reviews and comments
+
+    file_name = "tests/data/po/tutorial/en/tutorial_update.po"
+
+    # Update store as user
+    _update_from_upload_file(store, file_name, user=user)
+
+    # Add a suggestion
+    unit = store.units.first()
+    unit.add_suggestion('SUGGESTION', user=user)
+    return unit
+
+
+def _test_user_merged(unit, src, target):
+    # TODO: test reviews and comments
+
+    if src.id:
+        assert src.submitted.count() == 0
+        assert src.suggestions.count() == 0
+
+    assert target.submitted.first() == unit
+    assert target.suggestions.first() == unit.get_suggestions().first()
+
+
+@pytest.mark.django_db
+def test_merge_user(en_tutorial_po, member, member2):
+    """Test merging user to another user."""
+    unit = _create_submission_and_suggestion(en_tutorial_po, member)
+    accounts.utils.merge_user(member, member2)
+    _test_user_merged(unit, member, member2)
+
+
+@pytest.mark.django_db
+def test_delete_user(en_tutorial_po, member, nobody):
+    """Test default behaviour of User.delete - merge to nobody"""
+    unit = _create_submission_and_suggestion(en_tutorial_po, member)
+    member.delete()
+    _test_user_merged(unit, member, nobody)
+
+
+@pytest.mark.django_db
+def test_purge_user(en_tutorial_po, member):
+    """Test purging user - the django default behaviour."""
+    _create_submission_and_suggestion(en_tutorial_po, member)
+    member.delete(purge=True)
+    assert en_tutorial_po.units.count() == 0
```

Comment:
I don't know what this assert means. I was actually expecting some asserts that the translation/suggestion/review... counts for `nobody` didn't change.